tests/unit/specs/components/App.spec.js

```diff
@@ -30,6 +30,6 @@ describe('App.vue', () => {
     })
 
     it('Check if App.vue component renders without any errors', () => {
```

Comment:
Could you please fully remove all the tests that use `expect(wrapper.isVueInstance())` instead of replacing them. These tests are not very informative.
For App.spec.js specifically, could you also add a test that showLicenseUse is false in the beginning, but is set to true when the currentStepId is 7?

Reply:
@obulat I am working on App.spec.js